calabash-cucumber/scripts/.irbrc

```diff
@@ -47,3 +43,22 @@ extend Calabash::Cucumber::Operations
 def embed(x,y=nil,z=nil)
   puts "Screenshot at #{x}"
 end
+
+require "calabash-cucumber"
+
+def preferences
+  @preferences ||= Calabash::Cucumber::Preferences.new
+end
+
+def disable_usage_tracking
+  preferences.usage_tracking = "none"
+  puts "Calabash will not collect usage information."
+  "none"
+end
+
+def enable_usage_tracking(level="system_info")
+  preferences.usage_tracking = level
```

Comment:
Sure, this is a possibility, but I am not worried about it.  If a user reports a problem, we can address it later.